code/modules/mob/mob_movement.dm

```diff
@@ -161,6 +161,21 @@
 	moving = 1
 	move_delay = mob.movement_delay() + world.time
 
+	if(isliving(mob))
```

Comment:
This entire check is a bit odd, pretty sure we already have checks for lying down + crawling if you don't have legs, can we just modify that check and piggyback off it?